pkg/validators/helpers.go

```diff
@@ -1,3 +1,35 @@
 package validators
 
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
 func StringToPtr(s string) *string { return &s }
+
+func yamlToDynamicObj(yamlPath string) (unstructured.Unstructured, error) {
```

Comment:
nit: Would move to `utils` package as it's not really part of validators.

Reply:
Gotten rid of this entire file xD